src/components/expanded-state/UniqueTokenExpandedStateProps.ts

```diff
@@ -0,0 +1,40 @@
+export interface UniqueTokenExpandedStateProps {
+  asset: UniqueTokenExpandedStateAsset;
+  external: boolean;
+  lowResUrl: string;
+}
+
+// This structural type was created specifically for UniqueTokenExpandedState.
+// It should be superseded by a more authoritative UniqueToken type in the future.
+interface UniqueTokenExpandedStateAsset {
+  asset_contract: {
+    address: string;
+  };
+  background?: string | null;
+  collection: {
+    description?: string;
+    discord_url?: string;
+    external_url: string;
+    name: string;
+    slug: string;
+    twitter_username?: string;
+  };
+  currentPrice: string;
+  description?: string;
+  familyName: string;
+  familyImage: string;
+  id: string;
+  image_url: string;
+  isPoap?: boolean;
+  isSendable: boolean;
+  lastPrice?: string;
+  name: string;
+  permalink: string;
+  traits: {
+    trait_type: string;
+    value: string;
+  }[];
+  external_link: string;
+  uniqueId: number;
```

Comment:
this should be a string:  https://github.com/rainbow-me/rainbow/blob/559cd877db05454b8521f07fce3b5324a8d57c66/src/parsers/poap.js#L67

Reply:
It was clashing with this type: https://github.com/rainbow-me/rainbow/blob/644549e5f7bfb27f769430b90f7d0daefc982c1d/src/entities/uniqueAssets.ts#L3

Looks like that type is wrong though. I'll fix that one instead.